llpc/make/Makefile.llpc

```diff
@@ -231,4 +231,3 @@ else
 endif
 
 .FORCE:
-
```

Comment:
nit: unintended change?

Reply:
I think this is modified by my editoer, i'll rever it back to the original version.